### PR TITLE
fix preloadData-caused errors

### PR DESCRIPTION
**Pull request by juliuspfadt**

### CODE REVIEW

**Comment by juliuspfadt** [comment]
also the tests run on R 4.4.1... just not 4.4.2

File: renv.lock [hunk 1]
```diff
@@ -946,7 +946,7 @@
     },
     "knitr": {
       "Package": "knitr",
-      "Version": "1.49",
+      "Version": "1.48",
```
Comment (boutinb) [inline renv.lock:949]: Is this is on purpose, or the renv.lock should not be ipdated?

Comment (juliuspfadt) [reply renv.lock:949]: no idea. that is just the version I had on my computer

**Comment by boutinb** [comment]
The Data Library JASP files work fine now.
One issue still: if I use the ASRM - Mania file, and I do a 'Unidimensional Reliability' analysis (the frequentist one), then if I just select 2 variables, I get an error:
Screenshot 2024-11-25 at 10 58 04
If I add a 3rd variable, it works.

File: renv.lock (hunk 1 above)
Comment (boutinb) [reply renv.lock:949]: OK, we have tested with this renv.lock, so let is so. But better not to change this the next time if it is not necessary: the tests done so far were done with the old renv.lock.

Comment (juliuspfadt) [reply renv.lock:949]: I mean I have not changed it actively...

**Review by boutinb — APPROVED** [review]

**Pull request merged**